### ISSUE TITLE
Document rate limiting

### ISSUE THREAD

**Issue by joseprl89** [monzo/docs#66]
Hi, was testing the API to try and make an app out of it and hit the rate limit rather soon.

I'm unable to find any information on what are the limits or how long it takes to unblock you, guess that'd be nice to have in the docs 👍

**Comment by kiran94** [comment]
Has there been any update on this?

**Comment by dan-j** [comment]
I'd also like to know this. I was experimenting with the OAuth flow in Insomnia (which doesn't work because of the email magic-link which opens in a browser), and ended up hitting rate limits (429 responses). 

I'm fairly sure i only made ~30 requests in the space of 5 minutes, and still receiving 429's 10 minutes later.
